docs/ChangeLog-NEW.md

```diff
@@ -0,0 +1,21 @@
+ab073ce Moved action file to right location (#510)
```

Comment:
@pdhanoyasplunk Can you please remind me what the plan is for Changelog? Ideally we want to update the existing file